src/components/Attachments/AttachmentsSagas.js

```diff
@@ -0,0 +1,139 @@
+import { eventChannel, END } from 'redux-saga';
+import { all, put, call, fork, take, takeEvery } from 'redux-saga/effects';
+import { ResumableUploadToGoogleDrive } from 'libs/google-drive-uploader';
+import auth from 'auth';
+import * as Actions from './AttachmentsActions';
+
+const createUploaderCallback = emitter => (result, error) => {
+  if (error) {
+    emitter({
+      type: 'error',
+      error,
+    });
+    emitter(END);
+    return;
+  }
+
+  const handlersMap = {
+    initialize: () => emitter({ type: 'initialized' }),
+    getLocation: () => emitter({ type: 'getLocation' }),
+    start: () => emitter({ type: 'started' }),
+    Uploading: () =>
+      emitter({
+        type: 'uploading',
+        progress: result.progressNumber.current / result.progressNumber.end,
+      }),
+    Done: () => {
```

Comment:
Why `Uploading` and `Done` start with an uppercase letter, while all the other handlers start with a lowercase letter?

Reply:
legacy from cool uploading library